include/dlr_data_transform.h

```diff
@@ -7,61 +7,120 @@
 
 #include "dlr_common.h"
 
+#define NUM_DATE_TIME_COLS 7
```

Comment:
We could make this a private const member variable of `DateTimeTransformer` to be consistent with other constant values in other transformers.
`const int kNumDateTimeCols = 7;`